cli/main.rs

```diff
@@ -57,6 +58,7 @@ async fn main_impl() -> Result<()> {
                     temp_dir: None,
                     // Listen on an ephemeral port selected by the operating system
                     port: 0,
+                    access_token: Some(generate_lassie_access_token()),
```

Comment:
I tested this change manually by adding `console.log` to `runtime/js/fetch.js`, running `zinnia-dev run runtime/tests/js/ipfs_retrieval_tests.js` and observing output.